openmetadata-ui/src/main/resources/ui/public/locales/en-US/OpenMetadata/EmailConfiguration.md

```diff
@@ -1,68 +1,84 @@
 # Email Configuration
 
-Configure the SMTP Settings for sending emails.
+Openmetadata is able to send Emails on a various steps like Signup, Forgot Password , Reset Password, Change Event updates.
+<br/>
+Following configuration is needed to allow Openmetadata to send Emails.
 
 $$section
 
 ### Username $(id="username")
 
-SMTP server username.
+Username of the SMTP account.
 $$
 
 $$section
 
 ### Password $(id="password")
 
-SMTP server password.
+Password to be used with the username to send Emails.
 $$
 
 $$section
 
 ### Sender Email $(id="senderEmail")
 
-Mail of the sender.
+Email of the sender (can be same as `username`, but in Amazon SES this can different)
 $$
 
 $$section
 
 ### OpenMetadata URL $(id="openMetadataUrl")
 
-OpenMetadata server endpoint.
+Url of the Openmetadata Server, in case of Docker or K8s this needs to be the external Url used to access the UI.
 $$
 
 $$section
 
 ### Server Endpoint $(id="serverEndpoint")
 
-SMTP server endpoint.
+Endpoint of the SMTP server (Ex. smtp.gmail.com)
 $$
 
 $$section
 
 ### SMTP server port $(id="serverPort")
 
-SMTP Server Port.
+Port of the SMTP Server, this depends on the transportation strategy below.
+Following is the mapping between port and the transportation strategy.
+<br/>
```

Comment:
HI @mohityadav766 I don't think the markdown extension here supports HTML tags directly, or so I recall from @Sachin-chaurasiya.

If you want a line break, just add a physical break in the file and it should work.

Also, for example here, just using a list would be better:

```
- **SMTP:**- If SMTP port is 25 use this
- **SMTPS:**- If SMTP port is 465 use this
- **SMTP_TLS:**- If SMTP port is 587 use this
```

Reply:
@pmbrull above works fine I was suggested by @Sachin-chaurasiya to use </br>

Reply:
aaah ok. Then let's add it to the docs please @Sachin-chaurasiya !